components/infieldprogresscircle/stories/infieldprogresscircle.stories.js

```diff
@@ -0,0 +1,121 @@
+import { Sizes } from "@spectrum-css/preview/decorators";
+import { disableDefaultModes } from "@spectrum-css/preview/modes";
+import { size } from "@spectrum-css/preview/types";
+import { default as ProgressCircle } from "@spectrum-css/progresscircle/stories/progresscircle.stories.js";
+import packageJson from "../package.json";
```

Comment:
Oh! We need the metadata.json stuff.

```suggestion
import metadata from "../dist/metadata.json";
import packageJson from "../package.json";
```

Reply:
Something might be missing somewhere. It doesn't look like infield progress circle is correctly building the metadata.json in its `dist` file.